client/v2/internal/testpb/msg.proto

```diff
@@ -11,7 +11,13 @@ import "testpb/query.proto";
 
 service Msg {
   // Send a request and returns the request as a response.
-  rpc Send(MsgRequest) returns (MsgResponse);
+  rpc Send(MsgRequest) returns (MsgResponse) {
+    option (cosmos_proto.method_added_in) = "cosmos-sdk v0.50.0";
+  };
+
+  rpc Clawback(MsgClawbackRequest) returns (MsgClawbackResponse) {
+    option (cosmos_proto.method_added_in) = "cosmos-sdk v0.51.0";
```

Comment:
how is this added in 0.51 if we are still in 50.7?  These "added in" comments are not a great idea...